snippets/ztp_example-sno.yaml

```diff
@@ -43,8 +43,6 @@ spec:
     nodes:
       - hostName: "example-node1.example.com"
         role: "master"
```

Comment:
Please feel free to ignore this comment, it necessary, but should "master" be changed to another term based on https://redhat-documentation.github.io/supplementary-style-guide/#_master_and_slave . Maybe it's a coded name and it cannot be changed. Perhaps an example would be a good replacement, such as `<primary_role_name>`?

Reply:
I'll check with the SMEs, but I'll request a merge for this PR as it is out of scope for this issue. Good catch!